src/helpers/logCall.ts

```diff
@@ -30,9 +30,11 @@ export function logCall() {
 
             if (!this.noLogs && !this._overwrite) {
                 if (!key.includes("Chart") && !key.includes("save")) {
-                    const data =
-                        result instanceof SimpleData ? result._tempData : result
-                    showTable(data, this.nbTableItemsToLog)
+                    if (result instanceof SimpleData) {
+                        result.showTable(this.nbTableItemsToLog)
```

Comment:
This doesn't work, unfortunately. The result of a method is stored in **result**, which can be an array of objects or a returned Simple Data instance. If it's a Simple Data instance (it's the case our your line 34), the result of the method is stored inside **_tempData**. It's important because if **_overwrite** is **false**, we don't want to change **_data**, but still want to log **_tempData**. So maybe this line should be something like:
showTable(this.getTempData(), this.nbTableItemsToLog)
And maybe we should create a getTempData() method.